modules/ROOT/nav.adoc

```diff
@@ -89,7 +89,8 @@
  ** xref:n1ql:advanced.adoc[Advanced Features]
   *** xref:n1ql:n1ql-language-reference/flex-indexes.adoc[Flex Indexes]
   *** xref:n1ql:n1ql-language-reference/cost-based-optimizer.adoc[Cost-Based Optimizer]
-  *** xref:n1ql:n1ql-language-reference/transactions.adoc[N1QL Support for Couchbase Transactions]
+  *** xref:n1ql:n1ql-language-reference/transactions.adoc[]
+  *** xref:n1ql:n1ql-language-reference/transactions-walkthrough.adoc[]
```

Comment:
Stash this change and use it to address [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)